metadata.json

```diff
@@ -99,12 +99,12 @@
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": "3.5"
```

Comment:
ok i have adjustet the metadata.json to 3.6

2015-04-29 20:22 GMT+02:00 Zach Leslie <notifications@github.com>:

> In metadata.json
> <https://github.com/xaque208/puppet-unbound/pull/84#discussion_r29364658>:
>
> > @@ -99,12 +99,12 @@
> >      },
> >      {
> >        "name": "puppet",
> > -      "version_requirement": "3.x"
> > +      "version_requirement": "3.5"
>
> Ah, its the file resource in the concat module. I'd say lets just go to
> 3.6.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xaque208/puppet-unbound/pull/84/files#r29364658>.
>